angular/src/lib/popover/tests/__snapshots__/popover.component.spec.ts.snap

```diff
@@ -7,33 +7,41 @@ exports[`Popover Test should match snapshot 1`] = `
   <div
     class="cdk-overlay-connected-position-bounding-box"
     dir="ltr"
-    style="top: 0px; left: 0px; height: 0px; width: 0px; align-items: flex-start; justify-content: center;"
+    style="left: 0px; bottom: 0px; height: 0px; width: 0px; align-items: center; justify-content: flex-end;"
   >
     <div
       class="cdk-overlay-pane"
       id="cdk-overlay-0"
-      style="pointer-events: auto; position: static; transform: translateX(5px);"
```

Comment:
Translate is a good choice!!! Like it! Position (left/top/right/bottom) is not convenient to use in CDK overlay. The overlay bounding makes position complex.

Reply:
lol .. I didn't do this the cdk did